### DISCUSSION TITLE
Decision boundary plot for test data wrong

### DISCUSSION THREAD

**Discussion by ANOOPAANTO** [pycaret/pycaret#2018] [Q&A] [answered]
Class Report and confusion matrix good test data(holdout set) accuracy while Decision boundary plot shows very low  test data accuracy for the same model

**Comment by moezali1** [comment, marked as answer]
@ANOOPAANTO Decision boundary plot has nothing to do with the accuracy of a model. It's just a visual way of looking at the decision boundaries and you shouldn't take this plot very seriously if you have more than two features since the PCA will take away a lot of semantic details.
